src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -125,11 +125,6 @@ export class ItemByIdComponent implements OnDestroy {
         this.itemRouter.navigateTo(current.route, { page: action === ModeAction.StartEditing ? 'edit' : 'details' });
       }),
 
-      this.modeService.mode$.pipe(
-        filter(mode => [ Mode.Normal, Mode.Watching ].includes(mode)),
-        distinctUntilChanged(),
-      ).subscribe(() => this.reloadContent()),
-
```

Comment:
That part (which was not working anymore) is not required since the addition of [this code in item-datasource](https://github.com/France-ioi/AlgoreaFrontend/blob/master/src/app/modules/item/services/item-datasource.service.ts#L53)